### ISSUE TITLE
Webpack deprecation warning

### ISSUE THREAD

**Issue by marvinhagemeister** [vinaypuppal/preact-cli-plugin-netlify#8]
Thanks for an awesome plugin! We're planning to use it for our preactjs site :+1:

There is one warning that pops up on every build and I was able to narrow it down to `preact-cli-plugin-netlify`. For now it's harmless albeit a bit noisy. This is the output:

```bash
(node:12040) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
```

**Comment by vinaypuppal** [comment]
@marvinhagemeister Thanks for reporting this.

It's fixed in `v1.3.0`

**Comment by marvinhagemeister** [comment]
Wow that was quick! You're awesome! Thank you so much👍💯
